.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+# Related Issue
+- the issue goes here
```

Comment:
hmm, we can add a template like `fixes #[ISSUE NUMBER]` as they automatically close the linked issue if the PR is merged.

reference: https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords

Reply:
I added it for the same reason,but I thought the PR opener would write the keyword. But, mentioning **fixes** in the template itself is better. Thank you.

Reply:
Cool.